app/helpers/kbo-format.js

```diff
@@ -1,20 +1,6 @@
-const unformattedKboRegex = /[0-1][0-9]{9}/;
+import { helper } from '@ember/component/helper';
 
-export default function kboFormat(kbo) {
-  if (!(typeof kbo === 'string'))
-    throw new Error(
-      'kboFormat helper requires a single argument which has to be a string.',
-    );
-  if (!kbo.match(unformattedKboRegex)) {
-    console.warn(
-      `Kbo numbers in the backend are supposed to be formatted as a continuous sequence of 10 digits of which the first digit is either 0 or 1. Found "${kbo}". Bad data quality`,
-    );
-    return kbo;
-  }
-  const [one, two, three] = [
-    kbo.slice(0, 4),
-    kbo.slice(4, 7),
-    kbo.slice(7, 10),
-  ];
-  return `${one}.${two}.${three}`;
-}
+export default helper(function kboFormat([kbo]) {
+  if (!kbo || kbo.length !== 10) return '';
```

Comment:
Added errors (assertions) to the function. Don't just return empty string.
Check if parameter is string, if not error.
Check if length is ok, if not error. Even better. Use regex. `^\d{9}$`
You WANT the application to crash asap when something is wrong.
Especially helper functions for templates need type checks.